tempesta_fw/ss_skb.c

```diff
@@ -253,11 +250,11 @@ __skb_insert_after(struct sk_buff *skb, struct sk_buff *nskb)
 static inline void
 __skb_skblist_fixup(SsSkbList *skb_list)
```

Comment:
I believe we can and should remove `SsSkbList` now. Regarding `__skb_skblist_fixup()` the only place where we use it is `skb_fragment()` and it seems that we used the function just because we had no circular list, so the function seems also must be removed.

Reply:
`SsSkbList` is replaced with simple `struct sk_buff *skb_head` pointer to NULL-terminated list with additional tail pointer (`skb_head->prev`).